### ISSUE TITLE
Syntax Error after update to v4.1.0

### ISSUE THREAD

**Issue by wareczek** [lesshint/lesshint#425]
```
.test {
    height: @line-height-computed + @font-size-large;
}
```
```
An unknown error occurred when checking "/home/marek/dev/wins/zle-admin/app/imports/ui/forms.less", please file an issue with this info:
ParserError: Syntax Error at line: 1, column 23
    at Parser.error (/usr/lib/node_modules/lesshint/node_modules/postcss-values-parser/lib/parser.js:116:11)
    at Parser.operator (/usr/lib/node_modules/lesshint/node_modules/postcss-values-parser/lib/parser.js:172:18)
    at Parser.parseTokens (/usr/lib/node_modules/lesshint/node_modules/postcss-values-parser/lib/parser.js:234:14)
    at Parser.loop (/usr/lib/node_modules/lesshint/node_modules/postcss-values-parser/lib/parser.js:121:12)
    at Parser.parse (/usr/lib/node_modules/lesshint/node_modules/postcss-values-parser/lib/parser.js:50:17)
    at module.exports (/usr/lib/node_modules/lesshint/lib/utils/parse-value.js:12:35)
    at Object.urlFormatLinter [as lint] (/usr/lib/node_modules/lesshint/lib/linters/url_format.js:29:21)
    at linters.forEach (/usr/lib/node_modules/lesshint/lib/linter.js:94:38)
    at Array.forEach (native)
    at Linter.checkNode (/usr/lib/node_modules/lesshint/lib/linter.js:81:22)
```

**Comment by jwilsson** [comment]
Thanks for the report!

It looks like a issue in [postcss-values-parser](https://github.com/shellscape/postcss-values-parser), though. We'll need to track it down there.

**Comment by jwilsson** [comment]
No, wait. This is in our end. Updating `postcss-values-parser` (as was done in `4.1.0`) requires some more work. I'll revert it and tag a new version for now.

**Comment by jwilsson** [comment]
I hope this is fixed in `4.3.0`, if not please let me know!
